integreat_cms/cms/templates/linkcheck/links_by_filter.html

```diff
@@ -64,27 +67,43 @@ <h1 class="heading">
                     {% include 'linkcheck/link_list_row.html' %}
                 {% empty %}
                     <tr>
-                        <td colspan="6" class="px-4 py-3">
-                            {% trans 'No links detected yet.' %}
-                        </td>
+                        <td colspan="6" class="px-4 py-3">{% trans 'No links detected yet.' %}</td>
                     </tr>
                 {% endfor %}
             </tbody>
         </table>
+        {% if filtered_links %}
+            <div class="pt-2 px-2">
+                <div class="inline">
+                    <span class="text-gray-800 font-bold"
+                          data-broken-links-list-selection-count>0</span> <span class="text-gray-600">{% trans "Broken links selected" %}</span>
```

Comment:
```suggestion
                          data-broken-links-list-selection-count>0</span> <span class="text-gray-600">{% trans "Links selected" %}</span>
```
The link list is not only used for broken links, but also for valid links, unchecked links and ignored links...